mdk/src/main/java/com/example/examplemod/ExampleMod.java

```diff
@@ -46,7 +48,9 @@ private void setup(final FMLCommonSetupEvent event)
         LOGGER.info("DIRT BLOCK >> {}", Blocks.DIRT.getRegistryName());
     }
 
-    private void doClientStuff(final FMLClientSetupEvent event) {
+    @OnlyIn(Dist.CLIENT)
```

Comment:
NEVER use this in your code.

Reply:
Why? It's not deprecated.
Is only proxy are good way to separete client and server?

Reply:
How many times do I have to explain to people NOT to use ASM hacks when simple STANDARD java code will suffice?
Does it modify the classfile's bytecode? Don't use it!

Reply:
Ok, now DistExecutor are used for client-side event handler